action.yml

```diff
@@ -13,6 +13,9 @@ inputs:
   fail-build:
     description: 'Set to any value to cause build to fail upon failed anchore policy evaluation'
     required: false
+  include-packages:
```

Comment:
Can this be more specific? 'scan-application-packages' or 'include-application-packages'?